interface/src/app/download/download.tpl.html

```diff
@@ -29,52 +26,59 @@
-          </li>
-          <li class="list-group-item disabled">
-            3: <b>ml_data_source</b>
-          </li>
-          <!-- Optional fields -->
-          <li ng-repeat="item in annotations.included_types" as-sortable-item
-              class="list-group-item">
-            <div as-sortable-item-handle>
-              <i class="fa fa-trash fa-lg" aria-hidden="true"
-                 ng-click="del_type($index)"></i>
-              {{$index + 4}}: <b>{{item}}</b>
-            </div>
-          </li>
-        </ul>
+      <div class="col-xs-4">
+        <!-- Three fixed types -->
+        <table class="table table-bordered" style="margin-bottom: 0">
+          <thead>
+            <tr><th style="text-align: center">
+              Included Annotation Fields:
+            </th></tr>
+          </thead>
+          <tbody>
+            <tr ng-repeat="item in ctrl.fixedTypes">
+              <td>
+                {{$index + 1}}: {{item}}
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <!-- Optional types -->
+        <table class="table table-bordered table-hover" style="margin-top: 0">
+          <tbody ng-model="ctrl.includedTypes" as-sortable>
+            <tr ng-repeat="item in ctrl.includedTypes" as-sortable-item>
+              <td class="success" as-sortable-item-handle>
+                <i class="fa fa-trash fa-lg" aria-hidden="true"
+                   ng-click="ctrl.delType($index)"></i>
+                {{$index + 4}}: {{item}}
+              </td>
+            </tr>
+          </tbody>
+        </table>
       </div>
 
       <!-- Excluded types -->
-      <div class="col-xs-6">
-        <h4 class="text-center">
-          <b><font color="red">Excluded Annotation Fields:</font></b>
-        </h4>
-        <ul as-sortable ng-model="annotations.excluded_types"
-            class="list-group">
-          <li ng-repeat="item in annotations.excluded_types" as-sortable-item
-              class="list-group-item">
-            <div as-sortable-item-handle>
-              <i class="fa fa-undo fa-lg" ara-hidden="true"
-                 ng-click="add_type($index)"></i>
-              {{$index + 1}}: <b>{{item}}</b>
-            </div>
-          </li>
-        </ul>
+      <div class="col-xs-4">
+        <table class="table table-bordered table-hover">
+          <thead>
+            <tr><th style="text-align: center">
+              Excluded Annotation Fields:
+            </th></tr>
+          </thead>
+          <tbody ng-model="ctrl.excludedTypes" as-sortable>
+            <tr ng-repeat="item in ctrl.excludedTypes" as-sortable-item>
+              <td class="danger" as-sortable-item-handle>
+                <i class="fa fa-undo fa-lg" aria-hidden="true"
+                   ng-click="ctrl.addType($index)"></i>
+                {{$index + 1}}: {{item}}
+              </td>
+            </tr>
+          </tbody>
+        </table>
       </div>
+      <div class="col-xs-4"></div>
```

Comment:
This is a nice touch for large window sizes, but be careful of what will happen on smaller screens. There are better ways to get the columns to reflow naturally... have a look at http://getbootstrap.com/css/#grid for the overview of the Bootstrap layout system. I'm thinking maybe the "mobile and desktop" example (http://getbootstrap.com/css/#grid-example-mixed ) might have what you're looking for here.